packages/website/plugins/utils/rules.ts

```diff
@@ -17,24 +17,45 @@ export const eslintPluginDirectory = path.resolve(
 export const sourceUrlPrefix =
   'https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/';
 
-/**
- * @param withComment Whether to include a full comment note.
- * @remarks `withComment` can't be used inside a JSON object which is needed for eslintrc in the playground
- */
-export function getEslintrcString(
+function getRulesString(
   extendsBaseRuleName: string,
   stem: string,
   withComment: boolean,
 ): string {
-  return `{
-  "rules": {${
+  return `{${
     withComment
       ? '\n    // Note: you must disable the base rule as it can report incorrect errors'
       : ''
   }
     "${extendsBaseRuleName}": "off",
     "@typescript-eslint/${stem}": "error"
-  }
+  }`;
+}
+
+/**
+ * @param withComment Whether to include a full comment note.
+ * @remarks `withComment` can't be used inside a JSON object which is needed for eslintrc in the playground
+ */
+export function getEslintrcString(
+  extendsBaseRuleName: string,
+  stem: string,
+  withComment: boolean,
+): string {
+  return `{
+  "rules": ${getRulesString(extendsBaseRuleName, stem, withComment)}
+}`;
+}
+
+/**
+ * @param withComment Whether to include a full comment note.
+ */
+export function getEslintConfigString(
+  extendsBaseRuleName: string,
+  stem: string,
+  withComment: boolean,
+): string {
+  return `{
+  rules: ${getRulesString(extendsBaseRuleName, stem, withComment)}
```

Comment:
[Style] Both `getEslintrcString` and `getEslintConfigString` do very little other than wrap `getRulesString`. Is there any reason they can't be inlined?

Reply:
You're absolutely right! I was too focused on consolidating it.
I fixed it in [c2aad1c](https://github.com/typescript-eslint/typescript-eslint/pull/10164/commits/c2aad1c8b0c321cb51ae4a31dd0d91b96a0a8beb).